### PR TITLE
Fixing squid S2160 Subclasses that add fields should override "equals"

### PR DESCRIPTION
**Pull request by devFozgul**
This pull request is focused on resolving occurrences of Sonar rule 
 squid:S2160 - “Subclasses that add fields should override ""equals""”. 
This PR will remove 60min  TD.
 You can find more information about the issue here: 
 https://dev.eclipse.org/sonar/rules/show/squid:S2160
 Please let me know if you have any questions.
Fevzi Ozgul

### CODE REVIEW

File: core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/MultiShape2d.java [hunk 1]
```diff
@@ -222,6 +222,24 @@ public void shapeGeometryChange(Shape2d<?> shape) {
 			onContentGeometryChange();
 		}
 
+		public boolean equals(Object obj) {
+			if (!super.equals(obj)) {
```
Comment (gallandarakhneorg) [inline core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/MultiShape2d.java]: Should be:
```java
public boolean equals(Object obj) {
    return this.delegate.equals(obj);
}
```

Indeed, this class is a delegation design pattern and does no have any attributes except the delegate.

File: core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/MultiShape2d.java [hunk 2]
```diff
@@ -222,6 +222,24 @@ public void shapeGeometryChange(Shape2d<?> shape) {
...
+				return false;
+			}
+			final ListResponseModel lObj = (ListResponseModel) obj;
+			if (delegate.equals(lObj.delegate)) {
+				return delegate.equals(lObj.delegate);
+			}
+			return false;
+		}
+
+		public int hashCode() {
```
Comment (gallandarakhneorg) [inline core/math/src/main/java/org/arakhne/afc/math/geometry/d2/d/MultiShape2d.java]: Should be:
```java
public int hashCode() {
     return this.delegate.hashCode();
}
```

**Comment by devFozgul** [comment]
Failure in tests was due to some test classes which I had never touched. In those files ordering of import lines or unused imports were the reason for failure.

**Comment by gallandarakhneorg** [comment]
Tests are passing according to Travis.

**Comment by devFozgul** [comment]
Ok now. I corrected according to your direction then it passed.

**Pull request merged**